src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -66,10 +66,6 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('collectors.xml');
```

Comment:
@javiereguiluz the issue still occurs if `profiler` service is not defined, so `data_collector.security` require of `debug.security.access.decision_manager`